### ISSUE TITLE
Oder Placement Error: "BusinessRule ServiceMethodAllowedAtStore failed. Cause: nullOnStoreID"

### ISSUE THREAD

**Issue by broccolistalker** [RIAEvangelist/node-dominos-pizza-api#64]
I am having trouble placing an order. Has anyone else experienced this problem:
StatusItems:
   [ { Code: 'AutoAddedOrderId' },
     { Code: 'PriceInformationRemoved' },
     { Code: 'RuleExecutionError',
       Message: 'BusinessRule ServiceMethodAllowedForBuildingAtStore failed. Cause: nullOn StoreOrderID: ' } ],
This is present after attempting to validate the order.

**Comment by RIAEvangelist** [comment]
What store ID was this for?

**Comment by benhubsch** [comment]
I'm also having this issue for store ID 7764

**Comment by benhubsch** [comment]
It's actually happening for all stores.

**Comment by RIAEvangelist** [comment]
@benhubsch can you confirm this is an issue with the examples as well?

**Comment by benhubsch** [comment]
We ended up figuring out the issue. It had nothing to do with the store itself -- we were later able to place a store at that order after inspecting this file closely: https://github.com/bhberson/pizzadash/blob/master/exampleorder.json

**Comment by RIAEvangelist** [comment]
What was the issue afterall, if you remember?

**Comment by benhubsch** [comment]
Not entirely sure. It either had something to do with the fact that the Dominos didn't deliver to our location, or something about our types (strings vs. numbers) being wrong.

**Comment by clanderson87** [comment]
Getting something like this when I use `order.place(). order.validate()` returns success with `Status: 1`, but when place() is called on the exact same order, it returns a `Status: -1` and a statusItem of `{Code: "RuleExecutionError", Message: "BusinessRule BlacklistValidator failed. Cause: nullOn StoreOrderID: "}`

I've tried with the StoreID as a string and a number, both return the same message.

**Comment by clanderson87** [comment]
So after some trial and error I simply tried `order.StoreOrderID = order.storeID`. Both were 4 digit numbers. It worked and the order was successfully placed and correctly delivered. The `result` returned from order.place() returned some things I couldn't find in any of the docs: 

- `StoreOrderID: "2018-12-14#580XXX",` // notice this has changed.
- `StatusItems: [...{Code: "RuleExecutionError", Message: "BusinessRule BlacklistValidator Caused an exception. Cause: null Pulse Order successfully placed, so continuing business rules executionOn StoreOrderID: XXXX" }],`
-  `PulseOrderGuid: "a7983fd0-3386-4ba9-xxxx-xxxxxxxxxxxx"`

Don't know what any of those are mean, but I got pizza. I'll take it.
